rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/indices.open/10_basic.yml

```diff
@@ -125,6 +125,67 @@
   - match: { indices.index_1.closed: true }
   - match: { indices.index_2.closed: true }
   - match: { indices.index_3.closed: true }
+---
+"Close multiples indices using request body":
+  - skip:
+      features: ["allowed_warnings"]
+
+  - do:
+      indices.create:
+        index: index_1
+        body:
+          settings:
+            number_of_replicas: 0
+
+  - do:
+      indices.create:
+        index: index_2
+        body:
+          settings:
+            number_of_replicas: 0
+
+  - do:
+      indices.close:
+        body:
+          index:
+            - "index_1"
```

Comment:
Can we add a test case for sending a single string (ie `"index": "index_1") instead of a list as well? Thinking how clients implement path parameters for lists of values, we typically allow either a singular value or multiple values because "1" and ["1"] both serialize to the same path parameter value.